pkg/apis/workspaces/v1alpha2/containerComponent.go

```diff
@@ -42,6 +42,10 @@ type Container struct {
 	// +optional
 	Args []string `json:"args,omitempty" patchStrategy:"replace"`
 
+	// Toggles whether or not the project source code should
+	// be mounted in the component.
+	//
+	// Defaults to true for all component types except plugins and components that set `dedicatedPod` to true.
```

Comment:
GO-wise, you should change the type to `*bool` instead of `bool` if you want to change the default value. But that's slightly less convenient to work with in the GO source code since you have to check whether the field is `nil` before taking its value.

Reply:
Ah, that's a good point @davidfestal. I can update the type.